1906-12-10.xml

```diff
@@ -0,0 +1,2020 @@
+                                <cell><date when="1907-01-12">January 12th</date><date
+                                        when="1907-01-19">January 19th</date><date when="1907-01-26"
+                                        >January 26th</date></cell>
+                                <cell>arr. Naples</cell>
+                                <cell><date when="1907-01-15">January 15th</date><date
+                                        when="1907-01-22">January 22</date><date when="1907-01-29"
+                                        >January 29</date></cell>
+                            </row>
+                            <row>
+                                <cell>Leaves Alexandria</cell>
+                                <cell><date when="1907-02-02">February 2nd</date><date
+                                        when="1907-02-09">February 9th</date><date when="1907-02-16"
+                                        >February 16th</date><date when="1907-02-23">February
+                                        23rd</date></cell>
+                                <cell>arr. Naples</cell>
+                                <cell><date when="1907-02-05">February 5</date><date
+                                        when="1907-02-12">February 12</date><date when="1907-02-19"
+                                        >February 19</date><date when="1907-02-26">February
+                                        26</date></cell>
+                            </row>
+                            <row>
+                                <cell>Leaves Alexandria</cell>
+                                <cell><date when="1907-03-02">March 2nd</date><date
+                                        when="1907-03-09">March 9th</date><date when="1907-03-16"
+                                        >March 16th</date><date when="1907-03-23">March
+                                        23rd</date><date when="1907-03-30">March 30th</date></cell>
+                                <cell>arr. Naples</cell>
+                                <cell><date when="1907-03-05">March 5</date><date when="1907-03-12"
+                                        >March 12</date><date when="1907-03-19">March 19</date><date
+                                        when="1907-03-26">March 26</date></cell>
+                            </row>
+                            <row>
+                                <cell>Leaves Alexandria</cell>
+                                <cell><date when="1907-04-06">April 6th</date><date
+                                        when="1907-04-13">April 13th</date><date when="1907-04-27"
+                                        >April 27th</date></cell>
+                                <cell>arr. Naples</cell>
+                                <cell><date when="1907-04-02">April 2nd</date><date
+                                        when="1907-04-09">April 9th</date><date when="1907-04-16"
+                                        >April 16th</date><date 
```

Comment:
move the `</div>` to the end of the page--right now it's closing the page just after it opens